adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureRDD.scala

```diff
@@ -95,10 +102,117 @@ object FeatureRDD {
 
     FeatureRDD(rdd, sd)
   }
+
+  /**
+   * @param feature Feature to convert to GTF format.
+   * @return Returns this feature as a GTF line.
+   */
+  private[features] def toGtf(feature: Feature): String = {
+    def escape(entry: (Any, Any)): String = {
+      entry._1 + " \"" + entry._2 + "\""
+    }
+
+    val seqname = feature.getContigName
+    val source = Option(feature.getSource).getOrElse(".")
+    val featureType = Option(feature.getFeatureType).getOrElse(".")
+    val start = feature.getStart + 1 // GTF/GFF ranges are 1-based
+    val end = feature.getEnd // GTF/GFF ranges are closed
+    val score = Option(feature.getScore).getOrElse(".")
+    val strand = Features.asString(feature.getStrand)
+    val frame = Option(feature.getFrame).getOrElse(".")
+    val attributes = Features.gatherAttributes(feature).map(escape).mkString("; ")
+    List(seqname, source, featureType, start, end, score, strand, frame, attributes).mkString("\t")
+  }
+
+  /**
+   * @param feature Feature to write in IntervalList format.
+   * @return Feature as a one line interval list string.
+   */
+  private[rdd] def toInterval(feature: Feature): String = {
+    val sequenceName = feature.getContigName
+    val start = feature.getStart + 1 // IntervalList ranges are 1-based
+    val end = feature.getEnd // IntervalList ranges are closed
+    val strand = Features.asString(feature.getStrand, emptyUnknown = false)
+    val attributes = Features.gatherAttributes(feature)
```

Comment:
This goes back to shoving all attributes into the IntervalList format name column, right?  I saw that as an abuse to the format and generated a name with reasonable defaults, see https://github.com/bigdatagenomics/adam/blob/master/adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/Features.scala#L207.